tests/test_visitors/test_ast/test_statements/test_augmented_assign.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from wemake_python_styleguide.violations.consistency import (
+    AugmentedAssignPatternViolation,
+)
+from wemake_python_styleguide.visitors.ast.statements import (
+    AssignmentPatternsVisitor,
+)
+
+OPERATIONS = frozenset([
```

Comment:
`frozenset((`

We can make a new rule out of this! Do you want to create a request?

Reply:
Yes. We need a new rule) I'll create an issue. Also, have we rule about:
```
a in {1} # is not good, better: a == 1
```
?

Reply:
Awesome! Let's do it too! 

Two new rules, that's what I like 👍